raspbian/applications/vcgencmd.md

```diff
@@ -0,0 +1,131 @@
+| 3   | Soft temperature limit active |
+| 16  | Under-voltage has occurred |
+| 17  | Arm frequency capped has occurred |
+| 18  | Throttling has occurred |
+| 19  | Soft temperature limit has occurred |
+
+#### measure_temp
+
+Returns the temperature of the SoC as measured by the on board temperature sensor
+
+#### measure_clock [clock]
+
+This returns the current frequency of the specified clock. The options are:
+
+| clock | Description |
+|:-----:|-------------|
+| arm   | ARM cores |
+| core  | VC4 scaler cores |
+| H264  | H264 block |
+| isp   | Image system Pipeline |
+| v3d   | 3D block |
+| uart  | UART |
+| pwm   | PWM block (analogue audio output) | 
+| emmc  | SD card interface |
+| pixel | Pixel valve |
+| vec | Analogue video encoder |
+| hdmi | HDMI |
+| dpi | Display Peripheral Interface |
+
+e.g. `vcgencmd measure_clock arm`
+
+#### measure_volts [block]
+
+Displays the current voltages used by the specific block.
+
+| block | Description |
+|:-----:|-------------|
+| core | VC4 core voltage |
+| sdram_c | |
+| sdram_i | |
+| sdram_p | |
+
+#### otp_dump
+
+Displays the content of the One Time Programmable (OTP) memory, which is part of the SoC. These are 32 bit values, indexed from 8 to 64. See the [OTP bits page](../../../hardware/raspberrypi/otpbits.md) for more details.
+
+#### get_mem
+
+Reports on the amount of memory allocated to the ARM cores `vcgencmd get_mem arm` and the VC4 `vcgencmd get_mem gpu`.
+
+#### codec_enabled [type]
+
+Reports whether the specified CODEC type is enabled. Possible options for type are AGIF, FLAC, H263, H264, MJPA, MJPB, MJPG, **MPG2**, MPG4, MVC0, PCM, THRA, VORB, VP6, VP8, **WMV9**, **WVC1**. Those highlighted currently require a paid for licence. 
+
+#### get_config [type]
+
+This returns all the configuration items of the specified type that have been set in config.txt or by default. Possible values for type are **int, str**
+
+#### get_lcd_info
+
+Displays the resolution and colour depth of any attached display.
+
+#### mem_oom
+
+Displays statistics on any Out Of Memory events occuring in the VC4 memory space.
+
+#### mem_reloc_stats
+
+Displays statistics from the relocatable memory allocator on the VC4.
+
+#### read_ring_osc
+
+Returns the curent speed voltage and temperature of the ring oscillator.
+
+#### hdmi_timings
+
+Displays the current HDMI settings timings. See [Video Config](https://www.raspberrypi.org/documentation/configuration/config-txt/video.md) for details of the values returned. 
+
+#### dispmanx_list
+
+Dump a list of all dispmanx items currently being displayed.
+
+#### display_power
+
+Show or set the power the display. `vcgencmd display_power 0` will turn off power to the current display. `vcgencmd display_power 1` will turn on power to the display. If no parameter is set, this will display the current power state. The final parameter is an optional display ID, as returned by `tvservice -l`, which allows a specific display to be turned on or off.
```

Comment:
"Show or set the power the display." - something not right there :wink:

Reply:
Changed to "Show current display power state, or set the display power state"